### ISSUE TITLE
Knowledge tabs not updated until you change tabs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sunfall/endgame-singularity#85]
```
What steps will reproduce the problem?
1. Go to the knowledge screen, look at the contents of the techs tab
2. Research something
3. Go back to knowledge screen

What is the expected output? What do you see instead?

There are no new entries to reflect your current research. If you switch to
items or concepts and then switch back the list is updated. That should not
be required.

This bug exists in both 0.30 and current SVN.

What version of the product are you using? On what operating system?

SVN r944, Gentoo Linux
```

Original issue reported on code.google.com by `captains...@gmail.com` on 23 May 2009 at 11:30

**Comment by GoogleCodeExporter** [comment]
```
From Funnyman's post to singularity-dev:

> The file you're looking for will be code/screens/knowledge.py.  You'll
> probably want to add a show method (called whenever a dialog is
> shown), for which the basic form is this:

> def show(self):
>     # Set up anything special here.
>     # Preserve standard Dialog behavior. (visibility, close properly, etc.)
>     return super(KnowledgeScreen, self).show() 
```

Original comment by `justap...@gmail.com` on 29 Oct 2009 at 12:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `justap...@gmail.com` on 31 Oct 2009 at 5:02

Attachments:
* [b85.patch](https://storage.googleapis.com/google-code-attachments/endgame-singularity/issue-85/comment-2/b85.patch)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in git commit d3df694075, and already published in v0.30c
```

Original comment by `singular...@rodrigosilva.com` on 20 Dec 2012 at 2:52
* Changed state: **Fixed**
* Added labels: **Type-Defect**, **Priority-Low**, **Component-UI**, **Usability**, **OpSys-Linux**, **Milestone-0.30**
